pkg/controller/ms_test.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Can we move these tests to `machine_safety_test.go` file? As usually tests for a particular file are to be maintained in the `*_test.go` file for consistency.

Reply:
Sure, Prashanth. I will move the file content to `machine_safety_test.go` in both In-Tree and OOT.